Makefile

```diff
@@ -224,19 +253,19 @@ docker-login: ## Log in to a Docker registry.
 
 .PHONY: docker-build
 docker-build: test ## Build docker image with the manager.
-	docker build -t ${IMG} .
+	$(CONTAINER_COMMAND) build -t ${IMG} .
 
 .PHONY: docker-push
 docker-push: ## Push docker image with the manager.
-	docker push ${IMG}
+	$(CONTAINER_COMMAND) push  $(PODMAN_SKIP_TLS_VERIFY) ${IMG}
 
 .PHONY: bundle-build
 bundle-build: ## Build the bundle image.
-	docker build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
+	$(CONTAINER_COMMAND) build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
 
 .PHONY: bundle-push
-bundle-push: docker-login ## Push the bundle image.
```

Comment:
As the following command still might run docker, is it safe to remove the 'docker-login' dependency here?

Reply:
Both WLO and OLO do not have this dependency here. I am not sure why its here and if there is an actual need for it.